### PR TITLE
Task 8

### PR DESCRIPTION
**Pull request by zhenokin**
- [x] 1) Dockerfile is prepared, image is building. Image size is minimised to be less than 500Mb
image
- [x] 2) Dockerfile is optimized. Files that change more often and commands that depend on them should be included later, files and commands that change less should be at the top.
- [x] 3) Folders are added to .dockerignore, with explanations. At least 2 big directories should be excluded from build context. Elastic Beanstalk application is initialized.
- [x] 4) Environment is created and the app is deployed to the AWS cloud.
- [x] 5) FE application is updated with Cart API endpoint. 

FE PR: https://github.com/zhenokin/nodejs-aws-fe/pull/5
FE: https://dlnhu3eqgkqh.cloudfront.net/
Cart API: http://zhenokin-cart-api-dev.eu-west-1.elasticbeanstalk.com

### CODE REVIEW

**Comment by uladzimir-yeudakimovich** [comment]
image
need to using https

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Compiled output
```
Comment (kirill-ianchuk) [inline .dockerignore:1]: What about `.elasticbeanstalk`?

**Comment by zhenokin** [comment]
> image
> need to using https

Можешь подсказать как?

**Review by kirill-ianchuk — COMMENTED** [review]
> Можешь подсказать как?

One of the options is to use an API Gateway as a proxy: https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-create-api-as-simple-proxy-for-http.html
This solution is discussed in Discord in `task-8-aws` channel

File: .dockerignore (hunk 1 above)
Comment (zhenokin) [reply .dockerignore:1]: done

**Comment by zhenokin** [comment]
> image
> need to using https

fixed

**Pull request merged**